langchain4j-mistral-ai/src/test/java/dev/langchain4j/model/mistralai/MistralAiModerationModelIT.java

```diff
@@ -0,0 +1,33 @@
+package dev.langchain4j.model.mistralai;
+
+import dev.langchain4j.model.moderation.Moderation;
+import dev.langchain4j.model.moderation.ModerationModel;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+class MistralAiModerationModelIT {
+
+    ModerationModel model = new MistralAiModerationModel.Builder()
+        .apiKey("MISTRAL_AI_API_KEY")
```

Comment:
```suggestion
        .apiKey(System.getenv("MISTRAL_AI_API_KEY"))
```

Reply:
![image](https://github.com/user-attachments/assets/381a21a5-dca5-4f6d-9284-2bf218bccc85)


I added the key hardcoded, so assume the issue with System.getenv("MISTRAL_AI_API_KEY")

Reply:
Ah, got it!